rca/editorial/models.py

```diff
@@ -0,0 +1,116 @@
+from django.db import models
+from modelcluster.fields import ParentalKey
+from wagtail.admin.edit_handlers import (
+    FieldPanel,
+    InlinePanel,
+    MultiFieldPanel,
+    PageChooserPanel,
+)
+from wagtail.api import APIField
+from wagtail.core.fields import RichTextField
+from wagtail.images.edit_handlers import ImageChooserPanel
+
+from rca.utils.models import BasePage, RelatedPage
+
+
+class EditorialPageRelatedSchoolsAndResearchPages(RelatedPage):
+    source_page = ParentalKey(
+        "EditorialPage", related_name="related_schools_and_research_pages"
+    )
+    panels = [
+        PageChooserPanel("page", ["schools.SchoolPage", "research.ResearchCentrePage"])
+    ]
+
+    api_fields = [APIField("page")]
+
+
+class Author(models.Model):
+    name = models.CharField(max_length=128)
+
+    def __str__(self):
+        return self.name
+
+
+class EditorialPageAuthor(models.Model):
+    page = ParentalKey("EditorialPage", related_name="editorial_authors")
+    author = models.ForeignKey("Author", related_name="+", on_delete=models.CASCADE)
+    panels = [FieldPanel("author")]
+
+    def __str__(self):
+        return self.author.name
+
+
+class EditorialPage(BasePage):
+
+    intro_text = RichTextField(blank=True, max_length=255)
```

Comment:
Maybe the max_length does not need to be defined?

Reply:
Hm - looking at the designs I think the max length makes sense for the introduction field. But this should just be a text field, it doesn't need to be rich text as far as I know

Reply:
Changed to `intro_text = models.CharField(blank=True, max_length=255)`